filer/static/filer/js/base.js

```diff
@@ -43,29 +43,74 @@ var Cl = window.Cl || {};
             }
         }());
 
-        // mocking the action buttons to work in frontend UI, please do not review
+        // show counter if file is selected
+        (function () {
+            var table = $('.navigator-table').find('tr');
+            var actionCounter = $('.action-counter');
+            var actionSelect = $('.action-select, .column-checkbox');
+
+            actionSelect.each(function () {
+                $(this).on('click', function () {
+                    if (table.hasClass('selected')) {
+                        actionCounter.addClass('action-selected');
+                    } else {
+                        actionCounter.removeClass('action-selected');
+                    }
+                });
+            });
+        }());
+
         (function () {
             var dropdown = $('.js-actions-menu .dropdown-menu');
             var actionsSelect = $('.actions select[name="action"]');
             var actionsSelectOptions = actionsSelect.find('option');
             var actionsGo = $('.actions button[type="submit"]');
             var html = '';
-            var i = 0;
+            var actionDelete = $('.js-action-delete');
+            var actionCopy = $('.js-action-copy');
+            var actionMove = $('.js-action-move');
+            var valueDelete = 'delete_files_or_folders';
+            var valueCopy = 'copy_files_and_folders';
+            var valueMove = 'move_files_and_folders';
+
+            // triggers delete copy and move actions on separate buttons
+            function actionsButton(optionValue, actionButton) {
+                actionsSelectOptions.each(function () {
+                    if (this.value === optionValue) {
+                        actionButton.show();
+
+                        actionButton.on('click', function (e) {
+                            e.preventDefault();
+                            actionsSelect.val(optionValue).prop('selected', true);
+                            actionsGo.trigger('click');
+                        });
+                    }
+                });
+            }
+
+            actionsButton(valueDelete, actionDelete);
+            actionsButton(valueCopy, actionCopy);
+            actionsButton(valueMove, actionMove);
+
+            // mocking the action buttons to work in frontend UI
+            actionsSelectOptions.each(function (index) {
+                var className = '';
+                if (index !== 0) {
+                    if (this.value === valueDelete || this.value === valueCopy || this.value === valueMove) {
```

Comment:
The following looks better, IMO:
`if([valueDelete, valueCopy, valueMove].indexOf(this.value) > -1) {...}`